docs/contributing/3.0-development/app-layout.rst

```diff
@@ -0,0 +1,241 @@
+Pulp Platform Application Layout
+================================
+
+This documentation is for Pulp Platform developers. For Plugin development,
```

Comment:
Generally speaking, all the wrapping issues revolve around wanting to leave room to add links later just in case, and a general acknowledgement that it all gets wrapped properly when rendered. This also leaves room for future folks to come in here and tweak what they want without having to join/fold the whole paragraph again for a simple fix.

I was just eyeball-wrapping while writing this, there was no enforcement of a given wrap-width until 100 chars.